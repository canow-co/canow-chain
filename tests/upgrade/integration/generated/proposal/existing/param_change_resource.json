{
    "title": "Resource Param Change",
    "description": "This is a simple example of a param change - double the values, 99% burn factor",
    "type": "ParameterChange",
    "changes": [
        {
            "subspace": "resource",
            "key": "feeparams",
            "value": {
                "image": {
                    "denom": "zarx",
                    "amount": "20000000000"
                },
                "json": {
                    "denom": "zarx",
                    "amount": "5000000000"
                },
                "default": {
                    "denom": "zarx",
                    "amount": "10000000000"
                },
                "burn_factor": "0.990000000000000000"
            }
        }
    ],
    "deposit": "10000000000zarx"
}