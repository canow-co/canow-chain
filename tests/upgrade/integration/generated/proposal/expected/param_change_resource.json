{
    "image": {
        "denom": "zarx",
        "amount": "20000000000"
    },
    "json": {
        "denom": "zarx",
        "amount": "5000000000"
    },
    "default": {
        "denom": "zarx",
        "amount": "10000000000"
    },
    "burn_factor": "0.990000000000000000"
}